currencyfair.yaml

```
@@ -5536,6 +5536,18 @@ components:
         * `Cancelled: 3`
         * `Pending: 6`
         * `Returned: 10`
+    TransfersDetailsStatusEnum:
+      type: integer
+      enum:
+        - 1
+        - 2
+        - 3
+      description: |-
+        Possible values:
+        * `In progress: 1`
+        * `On hold:2`
```

Minor indentation fix for consistency
```suggestion
        * `On hold: 2`
```

---

addressed here https://github.com/CurrencyFair/xsapi/pull/15/commits/3d039ec8f5a96b1e8e45e583ee12098eafe39109